ex1/test_x_mix_drix.py

```diff
@@ -66,6 +71,12 @@ def test_out_of_bound_is_illegal_move(self):
         assert not board.is_move_legal(3, 0)
         assert not board.is_move_legal(0, 3)
 
+    def test_changing_value_only_one_value_change(self):
+        board = XOBoard.empty_board()
+
+        board.play_move(0, 0, X)
+        assert board.is_move_legal(1, 0)
+
 
 class TestGame(unittest.TestCase):
```

Comment:
Still looking forward for the completion of step 7 (add another test to check the functionality of `play_game`.